core/variant/type_info.h

```diff
@@ -129,6 +129,9 @@ MAKE_TYPE_INFO_WITH_META(int8_t, Variant::INT, GodotTypeInfo::METADATA_INT_IS_IN
 MAKE_TYPE_INFO_WITH_META(uint16_t, Variant::INT, GodotTypeInfo::METADATA_INT_IS_UINT16)
 MAKE_TYPE_INFO_WITH_META(int16_t, Variant::INT, GodotTypeInfo::METADATA_INT_IS_INT16)
 MAKE_TYPE_INFO_WITH_META(uint32_t, Variant::INT, GodotTypeInfo::METADATA_INT_IS_UINT32)
+#ifdef __OpenBSD__
+MAKE_TYPE_INFO_WITH_META(unsigned long, Variant::INT, GodotTypeInfo::METADATA_INT_IS_UINT32)
```

Comment:
This might result in duplicate template errors (should be checked on both 32-bit and 64-bit builds).

Usually, this type of errors can be fixed by simply adding `uint64_t` cast to the place where ambiguous int is assigned to the Variant variable.